web/auth/auth.go

```diff
@@ -224,9 +229,15 @@ func authorizeForm(c echo.Context) error {
 }
 
 // IsLoggedIn returns true if the context has a valid session cookie.
-func IsLoggedIn(c echo.Context) bool {
+func IsLoggedIn(c echo.Context) (bool, error) {
```

Comment:
I agree. `GetSession` should be sufficient in case we need the error when checking if `LoggedIn`.